src/main/resources/default/taglib/t/booleanSelect.html.pasta

```diff
@@ -1,30 +1,32 @@
-<i:arg name="id" type="String" default=""/>
+<i:arg name="id" type="String" default="@generateId('singleselect-%s')"/>
```

Comment:
```suggestion
<i:arg name="id" type="String" default="@generateId('booleanselect-%s')"/>
```

Reply:
Would be breaking if there is existing JS code depending on `singleselect-%s`.

I like the proper naming though and could adjust it accordingly.